app/controllers/json_api/event/levels_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+class JsonApi::Event::LevelsController < JsonApiController
+  def index
+    authorize!(:index, Event::Level)
+    super
+  end
+
+  def show
+    authorize!(:show, entry)
+    super
+  end
+
+  private
+
+  def entry
+    @entry ||= Event::Level.accessible_by(current_ability).find(params[:id])
```

Comment:
```suggestion
    @entry ||= Event::Level.find(params[:id])
```

Ich glaube das `#accessible_by` funktioniert aufgrund unserer Cancancan Implementation gar nicht. (Ausser bei `PersonReadables`). Das liegt daran dass unsere Ability DSL die Berechtigung mit Ruby code evaluiert und nicht mit SQL. Da das `#accessible_by` aber die SQL Methode benötigt bringt das glaub gar nichts

Reply:
hmm ok, ich habe dies einfach vom `app/controllers/json_api/external_invoices_controller.rb` und dem `people_managers_controller.rb` übernommen. wie soll man denn sonst die berechtigungen einschränken?